shapes/profile/app.shex

```diff
@@ -0,0 +1,29 @@
+PREFIX oidc: <http://www.w3.org/ns/solid/oidc#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schem: <http://schema.org/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<#id> EXTRA oidc:grant_types {
+  a schem:SoftwareApplication
+    // rdfs:comment  "A software application. (from Schema.org)"@en ;
+  oidc:redirect_uris IRI + 
```

Comment:
If there are two redirect URIs, what is the meaning?  Try either, or ty them all until one succeeds? Is hat defined in the spec?

Reply:
That is part of the OAuth2 protocol. An app will make use of one particular `redirect_uri`, which needs to have been previously registered (e.g. via this mechanism). An OAuth client registration may have multiple redirect URIs because it may handle several different client applications.